priv/repo/migrations/20200209041307_create_transactions.exs

```diff
@@ -0,0 +1,20 @@
+defmodule MonisApp.Repo.Migrations.CreateTransactions do
+  use Ecto.Migration
+
+  def change do
+    create table(:transactions, primary_key: false) do
+      add :id, :binary_id, primary_key: true
+      add :comment, :string
+      add :payee, :string, null: false
+      add :value, :integer, null: false
+      add :transaction_date, :date, null: false
+      add :category_id, references(:category, on_delete: :nothing, type: :binary_id), null: false
```

Comment:
I thinks this could cause problems, if a category is deleted and its reference is not changed to something, any requests to this category will break, such as showing the category in the frontend. Maybe a better strategy is to set `on_delete` to `:nilify_all` so that transactions of a deleted category are "orphaned", then change the scheme to be able to return a `nil` `category_id`, which would cause the frontend to treat this special case differently

Reply:
Furthermore, I think that deleting is not the best way to manage categories, we could simply set them as `disabled`, which would imply that no new transactions can be created under that category, but existing ones will remain there

Reply:
> I thinks this could cause problems, if a category is deleted and its reference is not changed to something, any requests to this category will break, such as showing the category in the frontend.

This won't happen because postgres won't let you delete a category with related transactions, as it would break database consistency

> Furthermore, I think that deleting is not the best way to manage categories, we could simply set them as disabled, which would imply that no new transactions can be created under that category, but existing ones will remain there

I agree, but I think this should be addressed in another PR as it's only related to category. What do you think?